npm/webpack-dev-server/test/makeWebpackConfig.spec.ts

```diff
@@ -64,13 +69,16 @@ describe('makeWebpackConfig', () => {
             overlay: true, // This will be overridden by makeWebpackConfig.ts
           },
         },
+        optimization: {
+          emitOnErrors: false, // This will be overridden by makeWebpackConfig.ts
+        },
       },
       devServerEvents: new EventEmitter(),
     }
     const actual = await makeWebpackConfig({
       devServerConfig,
       sourceWebpackModulesResult: createModuleMatrixResult({
-        webpack: 4,
+        webpack: 5,
```

Comment:
This should be 5, not 4, since `webpack-dev-server` v4 uses Webpack 5. Confused yet?